models/Asset/Image/Thumbnail/Processor.php

```diff
@@ -268,7 +268,9 @@ public static function process(Asset $asset, Config $config, $fileSystemPath = n
                 }
 
                 if (is_resource($fileSystemPath)) {
+                    $fileSystemPathStream = $fileSystemPath;
                     $fileSystemPath = self::getLocalFileFromStream($fileSystemPath);
+                    fclose($fileSystemPathStream);
```

Comment:
```suggestion
                    @fclose($fileSystemPathStream);
```
just to be on safe side..